pkg/api/api.go

```diff
@@ -54,7 +54,7 @@ type CredentialReader interface {
 	// Get retrieves a VC.
 	Get(id string) (*verifiable.Credential, error)
 	// GetAll retrieves all VCs.
-	GetAll() ([]verifiable.Credential, error)
+	GetAll() ([]*verifiable.Credential, error)
```

Comment:
In aries it is common practice to use []*verifiable.Credential. Also function like ParseCredential returns *verifiable.Credential. It will require some extra code to copy the content of *verifiable.Credential into verifiable.Credential